workflow/workflow-demos-test/src_test/test/bpm/TestProcurementRequest.java

```diff
@@ -44,25 +42,27 @@
     BpmElement VERIFY_REQUEST_MANAGER = PROCUREMENT_PROCESS.element().name().contains("by Manager");
     BpmElement ACCEPT_REQUEST = PROCUREMENT_PROCESS.elementName("Accept Request");
   }
+  
+  private static final String MANAGER = "Manager";
+  private static final String TEAMLEADER = "Teamleader";
+  private static final String EXECUTIVE_MANAGER = "Executive Manager";
 
   /**
    * Manager and Team Leader do not verify
    */
   @Test
   void approvalTask_notVerified_byBoth(BpmClient bpmClient)
   {
-    List<ITask> verifyTasks = createProcurrementRequest(bpmClient);
+    TaskSelector verifyTasks = createProcurrementRequest(bpmClient);
```

Comment:
I'm not sure if this is easy to understand for new dummy users. I think it is better to return an exection result here. So that it is clear where this task selector comes from on top level.